_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -388,6 +388,8 @@ function jetpack_current_user_data() {
 	global $current_user;
 	$is_master_user = $current_user->ID == Jetpack_Options::get_option( 'master_user' );
 	$dotcom_data    = Jetpack::get_connected_user_data();
+	// Add connected user gravatar to the returned dotcom_data
+	$dotcom_data[ 'avatar'] = Jetpack::get_avatar_url( $dotcom_data[ 'email' ] )[ 0 ];
```

Comment:
The result of `Jetpack::get_avatar_url` should be added to a temp variable so it's PHP 5.2 compatible
http://php.net/manual/en/language.types.array.php#example-104
`As of PHP 5.4 it is possible to array dereference the result of a function or method call directly. Before it was only possible using a temporary variable.`